runelite-client/src/main/java/net/runelite/client/plugins/slayer/SlayerConfig.java

```diff
@@ -207,4 +207,22 @@ default int slaughter()
 		description = ""
 	)
 	void slaughter(int slaughter);
+
+	@ConfigItem(
+			keyName = "doubleTroubleEnabled",
+			name = "",
+			description = "",
+			hidden = true
+	)
+	default boolean doubleTroubleEnabled()
```

Comment:
Is the VarPlayer only updated when the rewards interface is open? If not, all this config stuff is useless, just use the boolean you're setting in VarbitChanged.